### PR TITLE
Resnet50 || ImageNet || GammaLab 

### PR DESCRIPTION
**Pull request by GammaLab-HPC**

### CODE REVIEW

File: ImageNet/inference/GammaLab_resnet50_PingAnCloud_Caffe.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "v1.0",
+    "author": "PingAn GammaLab & PingAn Cloud team",
+    "authorEmail": "",
```
Comment (codyaustun) [inline ImageNet/inference/GammaLab_resnet50_PingAnCloud_Caffe.json]: Please add at least one author email address.

**Review by codyaustun — CHANGES_REQUESTED** [review]

File: ImageNet/inference/GammaLab_resnet50_PingAnCloud_Caffe.json (hunk 1 above)
Comment (GammaLab-HPC) [reply ImageNet/inference/GammaLab_resnet50_PingAnCloud_Caffe.json]: Tanks, Author email address has been added.

**Review by codyaustun — APPROVED** [review]

File: ImageNet/inference/GammaLab_resnet50_PingAnCloud_Caffe.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "v1.0",
+    "author": "PingAn GammaLab & PingAn Cloud team",
+    "authorEmail": "wuxiabao@hotmail.com ，hncstjg@163.com",
+    "framework": "Caffe",
```
Comment (codyaustun) [inline ImageNet/inference/GammaLab_resnet50_PingAnCloud_Caffe.json]: Can you add the version number for Caffe?

**Review by codyaustun — COMMENTED** [review]
Looks good overall, but it would be good to add the version number for Caffe

File: ImageNet/inference/GammaLab_resnet50_PingAnCloud_Caffe.json (hunk 2 above)
Comment (GammaLab-HPC) [reply ImageNet/inference/GammaLab_resnet50_PingAnCloud_Caffe.json]: OK，we have added the version，and we have also updated a new result of inference latency

Comment (codyaustun) [reply ImageNet/inference/GammaLab_resnet50_PingAnCloud_Caffe.json]: Thanks!

**Pull request merged**